src/app/presales/components/pre-sale-section/index.tsx

```diff
@@ -1,22 +1,30 @@
 'use client'
 
 import { useDisclosure } from '@chakra-ui/react'
-import Link from 'next/link'
 import { useRouter } from 'next/navigation'
+import { useState } from 'react'
 
 import { preSaleTokens } from '../../constants'
-import { PreSaleStatus } from '../../types'
+import { PreSaleStatus, PreSaleToken } from '../../types'
 import { PreSalePopup } from '../popup'
 import { PreSaleTokenCard } from '../pre-sale-token-card'
 
+import { Disclaimer } from './disclaimer'
+
 export function PreSaleSection() {
+  const [presaleToken, setPresaleToken] = useState<PreSaleToken>(
+    preSaleTokens[1],
```

Comment:
Is this controlling which presaleToken is defaulted when the widget opens? If so, should it be updated to `presaleTokens[0]` following the re-ordering?